FAILED: ClusterSingletonManagerLeave2 (release-2.5)

https://jenkins.akka.io:8498/job/akka-release-2.5/175/consoleFull

```
[info] [JVM-3] - must hand-over to new instance (on node 'third', class akka.cluster.singleton.ClusterSingletonManagerLeave2MultiJvmNode3) *** FAILED ***
[info] [JVM-3]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for preStart
[info] [JVM-3]   at scala.Predef$.assert(Predef.scala:223)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg_internal(TestKit.scala:418)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg(TestKit.scala:395)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg$(TestKit.scala:395)
[info] [JVM-3]   at akka.testkit.TestKit.expectMsg(TestKit.scala:923)
[info] [JVM-3]   at akka.cluster.singleton.ClusterSingletonManagerLeave2Spec.$anonfun$new$27(ClusterSingletonManagerLeave2Spec.scala:181)
[info] [JVM-3]   at akka.remote.testkit.MultiNodeSpec.runOn(MultiNodeSpec.scala:383)
[info] [JVM-3]   at akka.cluster.singleton.ClusterSingletonManagerLeave2Spec.$anonfun$new$2(ClusterSingletonManagerLeave2Spec.scala:181)
[info] [JVM-3]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[info] [JVM-3]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
```

--- a/akka-cluster-tools/src/multi-jvm/scala/akka/cluster/singleton/ClusterSingletonManagerLeave2Spec.scala
+++ b/akka-cluster-tools/src/multi-jvm/scala/akka/cluster/singleton/ClusterSingletonManagerLeave2Spec.scala
@@ -178,7 +178,7 @@ class ClusterSingletonManagerLeave2Spec
       enterBarrier("stopped")
 
       runOn(third) {
-        expectMsg("preStart")
+        expectMsg(10.seconds, "preStart")
       }
       enterBarrier("third-started")
 

